R/build-articles.R

```diff
@@ -94,31 +112,64 @@ build_articles <- function(pkg = ".", path = "docs/articles", depth = 1L,
 render_rmd <- function(pkg,
                        input,
                        output_file,
+                       rmd_times,
+                       html_times,
                        strip_header = FALSE,
                        data = list(),
                        toc = TRUE,
                        depth = 1L,
                        encoding = "UTF-8",
-                       quiet = TRUE) {
-  message("Building article '", output_file, "'")
+                       quiet = TRUE,
+                       lazy = TRUE) {
+  if (missing(html_times))
```

Comment:
This is rather convoluted. I think it should just be something like `!lazy || out_of_date(input, output_file)`